src/Tinker.jl

```diff
@@ -12,21 +25,11 @@ mutable struct ImageContext{T}
     pandrag::Signal{Bool}
     zoomclick::Signal{Bool}
     rectselect::Signal{Bool}
+    shape::Signal{Rectangle} # Holds selection outline
```

Comment:
It should! However, it doesn't seem to like it when I say so. I had a similar problem with typing the rectview field (should be the next one after shape). I think it ought to be a signal of type AbstractArray, but again it throws an error when I say so. Do you think the fact that they're signals is causing the problem somehow?